src/fides/lib/oauth/roles.py

```diff
@@ -94,6 +96,8 @@ class RoleRegistryEnum(Enum):
     DATA_SUBJECT_READ,
     DATA_USE_READ,
     EVALUATION_READ,
+    MASKING_EXEC,
```

Comment:
I added these to viewer scopes, but please check me on if this makes sense! Does the masking endpoint just return a masked value or does it go ahead and mask something with a side effect?

Reply:
Masking endpoint just returns a masked value - no side effects in the system.  It's a weird one, this endpoint. I keep changing my mind as to whether a "viewer" should have this scope or not. I mean it doesn't hurt anything. Just not an action I associate with "viewer"

Reply:
Fine with me! I think it's unlikely they'll know about this :)